packs/feats/arcane-fists.json

```diff
@@ -64,6 +44,18 @@
                 "mode": "add",
                 "property": "weapon-traits",
                 "value": "arcane"
+            },
+            {
+                "key": "CriticalSpecialization",
+                "predicate": [
+                    "item:group:brawling",
```

Comment:
```suggestion
                    "item:category:unarmed",
                    "item:group:brawling",
```

Reply:
Are you certain? The description says "At 5th level, you also gain the critical specialization effects of unarmed attacks in the brawling group **and weapons** in the brawling group."

Reply:
Nope, my mistake: ignore